oshi-core/src/main/java/oshi/hardware/common/AbstractGlobalMemory.java

```diff
@@ -37,4 +38,15 @@
     protected long memAvailable = -1L;
     protected long pageSize = -1L;
     protected VirtualMemory virtualMemory;
+    
+    @Override
+    public String toString() {
+    	StringBuilder sb = new StringBuilder();
+    	sb.append("Memory: ");
```

Comment:
Let's leave this "Memory" header off the toString and keep the text in the SystemInfoTest.  And we might want to add the word "Available:" here instead.

Reply:
Added change for this with "Available" text indented and did the same for "Swap"